crates/core/component/dex/src/swap/view.rs

```diff
@@ -55,18 +65,21 @@ impl From<SwapView> for pb::SwapView {
             SwapView::Visible {
                 swap,
                 swap_plaintext,
+                output_1,
+                output_2,
+                claim_tx,
             } => Self {
                 swap_view: Some(sv::SwapView::Visible(sv::Visible {
                     swap: Some(swap.into()),
                     swap_plaintext: Some(swap_plaintext.into()),
-                    // Swap claim crossreferencing is not yet supported in the Rust stack.
-                    claim_tx: None,
+                    output_1,
+                    output_2,
+                    claim_tx,
+
                     // These fields are also not yet supported in the Rust stack.
                     asset_1_metadata: None,
                     asset_2_metadata: None,
                     batch_swap_output_data: None,
```

Comment:
Don't we want this metadata too? We'd need it to render the assets (symbols, icons, etc) for a `Swap` without a corresponding `SwapClaim` right?

Reply:
xref https://github.com/penumbra-zone/penumbra/pull/4035#discussion_r1535134495